packages/app/src/specs/SpecsListBanners.vue

```diff
@@ -228,10 +232,10 @@ const showConnectBanner = ref(false)
 const showCreateOrganizationBanner = ref(false)
 const showLoginBanner = ref(false)
 
-const hasRecordBannerBeenShown = ref(true)
-const hasConnectBannerBeenShown = ref(true)
-const hasCreateOrganizationBannerBeenShown = ref(true)
-const hasLoginBannerBeenShown = ref(true)
+const hasRecordBannerBeenShown = ref(false)
```

Comment:
I am looking forward to moving these into the store. It will massively simplify things... eg instead of the huge v-if/else, and we emit an event to update these, the components themselves could be responsible for updating this. Inside the `<RecordBanner />` we could just do `store.hasRecordBannerBeenShown = true`, right now the `<SpecsListBanner />` IS the store.